src/build.ts

```diff
@@ -9,7 +9,8 @@ import { walk } from "https://deno.land/std@0.177.0/fs/walk.ts";
 import { create_handler } from "./server.ts";
 import { globToRegExp } from "https://deno.land/std@0.182.0/path/glob.ts";
 import { copy } from "https://deno.land/std@0.179.0/fs/copy.ts";
-import { normalize } from "https://deno.land/std@0.177.0/path/mod.ts";
+import { expandGlob } from "https://deno.land/std@0.179.0/fs/mod.ts";
+import { dirname, normalize } from "https://deno.land/std@0.177.0/path/mod.ts";
```

Comment:
Could we stick with a single version of the standard lib?

```suggestion
import { expandGlob } from "https://deno.land/std@0.177.0/fs/mod.ts";
import { dirname, normalize } from "https://deno.land/std@0.177.0/path/mod.ts";
```